openerp/http.py

```diff
@@ -542,7 +543,39 @@ def dispatch(self):
         if self.jsonp_handler:
             return self.jsonp_handler()
         try:
+            rpc_request = logging.getLogger(__name__ + '.rpc.request')
```

Comment:
Could we define the loggers at the beginning of the file, to avoid duplicating this code with the one in `dispatch_rpc()`?